modules/apps/user/user-test/src/testIntegration/java/com/liferay/user/service/permission/test/UserPermissionTest.java

```diff
@@ -117,6 +130,58 @@ public void testOrganizationActionIds() throws Exception {
 				permissionChecker, _user2.getUserId(), ActionKeys.UPDATE));
 	}
 
+	@Test
+	public void testUpdateFieldPermissionWithInheritedRole() throws Exception {
```

Comment:
hey @wanderlast please evaluate to refactor this test using the `ContextUserReplace` if possible). It should simplify the test